documentation/documentation.gradle

```diff
@@ -77,7 +77,7 @@ asciidoctor {
 				'mainDir': project.sourceSets.main.java.srcDirs[0],
 				'testDir': project.sourceSets.test.java.srcDirs[0],
 				'testResourcesDir': project.sourceSets.test.resources.srcDirs[0],
-				'source-highlighter': 'coderay', // TODO switch to 'rouge' once supported by the html5 backend
+				'source-highlighter': 'rouge',
```

Comment:
Thanks for chiming in, @mojavelinux! 

What does the `@` do at the end of `coderay@`?